cmd/record.go

```diff
@@ -206,6 +221,8 @@ func (r *Record) GetCmd() *cobra.Command {
 
 	recordCmd.Flags().Uint64P("delay", "d", 5, "User provided time to run its application")
 
+	recordCmd.Flags().Uint64P("buildDelay", "", 30, "User provided time to wait docker container build")
```

Comment:
Duration would be a better datatype for inputting delay. Some docker builds can take minutes to startup. You can input it like that using Duration. Please make sure to change it everywhere. You can refer this: https://pkg.go.dev/time

Reply:
You won't be able to configure time like '30m5s'. Even if you do, that would be unnecessary complexity into this

Reply:
> You won't be able to configure time like '30m5s'. Even if you do, that would be unnecessary complexity into this

That's right, I'll go ahead and set the buildDelay type to time.Duration and specify the parameter's default unit to be seconds. Then I'll modify the relevant directive help